assets/src/edit-story/components/panels/layer/panel.js

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { Panel, PanelTitle, PanelContent } from '../panel';
+import { DEFAULT_LAYERS_VISIBLE, LAYER_HEIGHT } from './constants';
+import Layer from './layer';
+import useLayers from './useLayers';
+
+const LayerList = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+`;
+
+function LayerPanel() {
+  const layers = useLayers();
+
+  return (
+    <Panel name="layers" initialHeight={DEFAULT_LAYERS_VISIBLE * LAYER_HEIGHT}>
+      <PanelTitle isPrimary isResizable>
+        {__('Layers', 'web-stories')}
+      </PanelTitle>
+
+      <PanelContent isScrollable padding={'0'}>
+        <LayerList>
+          {layers.map((element) => (
+            <Layer key={element.id} element={element} />
```

Comment:
A11y: Either the `Layer` component should be a `li`, or it should be wrapped by one here.

Reply:
But should it actually? `role="listitem"` isn't an interactive element. I think `role="option"` wrapped in a `role="listbox"` is a better a11y pattern here and that's the one I'm currently going for in my a11y extension of this PR.

Reply:
But can we postpone the a11y stuff in your 3 comments for a future PR? I want to do it properly and I don't see a huge need for a temporary fix in this one.

Reply:
Please see #270.

Reply:
Sure, we can do that. The improved labels would also be added in #270? Or separately?